### PR TITLE
Mention the OData records limit

### PR DESCRIPTION
**Pull request by Aquassaut**

### CODE REVIEW

File: source/includes/odata/odata_basics.md [hunk 1]
```diff
@@ -271,6 +271,8 @@ The server will ignore `$skip` results and then return the first `$top` items.
 
 When paging is applied, a link to the next results will be added at the end of the payload.
 
+Please note that the OData API does not allow a result window larger than 10000.
```
Comment (thoumasd) [inline source/includes/odata/odata_basics.md]: I think we could be more explicit: 

```Please note that the OData API does not allow the resultset to be larger than 10,000 results.  So, for instance, you won't be able to request more than 100 pages of 100 results. To retrieve the whole content of datasets larger than 10000 records, you should use the OpenDataSoft Download API.```

Comment (Aquassaut) [reply source/includes/odata/odata_basics.md]: OK on the principle, but it's a bit weird for OData semantics though: if I ask the server to give me the $top 1000 results after $skipping 1000000, I expect the resultset size to be of 1000

**Comment by thoumasd** [comment]
Yes, you are right, we shouldn’t use the term resultset.

**Comment by Aquassaut** [comment]
I believe this version will be best

**Pull request merged**